dot/sync/test_helpers.go

```diff
@@ -88,3 +87,5 @@ func BuildBlock(t *testing.T, instance runtime.Instance, parent *types.Header, e
 		Body:   body,
 	}
 }
+
+const defaultSlotDuration = 6 * time.Second
```

Comment:
This is only used in `chain_sync_test.go` so move in that file please.
That way it will leave the production code Go API unaffected by test-only constants.